bin/express

```diff
@@ -185,6 +190,12 @@ function createApplication(app_name, path) {
     // Template support
     app = app.replace('{views}', program.template);
 
+    // Conditionally remove dust.js template code
+    if (program.template !== 'dust') {
+      app = app.replace("var adaro = require('adaro');\n", '');
```

Comment:
This is going to be fragile. Can it be done similar to how the CSS stuff is done, where this is added, not removed?